src/conversions/std/string.rs

```diff
@@ -233,7 +233,152 @@ impl FromPyObject<'_, '_> for char {
     }
 }
 
-#[cfg(test)]
+// FFI conversions for CStr and CString
+
+impl<'py> IntoPyObject<'py> for &std::ffi::CStr {
+    type Target = PyString;
+    type Output = Bound<'py, Self::Target>;
+    type Error = Infallible;
+
+    #[cfg(feature = "experimental-inspect")]
+    const OUTPUT_TYPE: &'static str = "str";
+
+    #[inline]
+    fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
+        // Convert cstr to &str, it's is safe because cstr is guaranteed to be valid UTF-8
```

Comment:
This is not true, a `CStr` can safely be constructed from any `&[u8]` without interior null bytes an thus may _not_ contain valid utf8.

Instead of `unwrap`ping this should use `Utf8Error` as its `Error` type, to forward the error to the caller.